app/styles/main.scss

```diff
@@ -403,6 +403,13 @@ body {
         padding: 10px;
     }
 
+    .ad-mob-page-container {
+        padding-top: 0;
+        padding-left: 10px;
```

Comment:
BTW something for the future: probably app should start using responsive vh units instead of px